swagger/public_v1/template.yml

```diff
@@ -435,6 +435,66 @@ components:
             $ref: "#/components/schemas/LocationResource"
         jsonapi:
           $ref: "#/components/schemas/JSONAPI"
+    LocationStatus:
```

Comment:
We've used the suffix `Resource` on other resources to try to make it clear ... can we do that here for consistency? Not averse to changing the naming, but let's do it across the board.

Reply:
yep, done